flocker/control/httpapi.py

```diff
@@ -1079,3 +1159,21 @@ def create_api_service(persistence_service, cluster_state_service, endpoint,
             Site(api_root)
         )
     )
+
+
+def lease_to_json(lease, now):
```

Comment:
Kind of a misleading function name, since it doesn't actually return JSON. The containers equivalent is called `container_configuration_response`.